op-mode-definitions/reset-wireguard.xml.in

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="reset">
+    <children>
+      <node name="wireguard">
+        <properties>
+          <help>Reset WireGuard Peers</help>
+        </properties>
+        <children>
+          <node name="interface">
+            <properties>
+              <help>Reset specific WireGuard interface</help>
+            </properties>
+            <children>
+              <tagNode name="">
```

Comment:
The tag node should be named `interface` instead of the regular node and this empty tag node.